mock/py/mock.py

```diff
@@ -40,15 +38,16 @@
        mock [options] --yum-cmd arguments
 """
 
+from __future__ import print_function
```

Comment:
Do we still need this? Even EPEL7 now uses python3.

Reply:
Probably not.  But since I wanted to test that one, I already merged that commit in a separate PR.  Sorry...